### ISSUE TITLE
ENM ontology typos

### ISSUE THREAD

**Issue by vedina** [enanomapper/ontologies#282]
http://purl.enanomapper.org/onto/ENM_8000110  - "valance band" - typo - "valence band"

ENM_8000102 - mullikens   electronegativity xc - > Mulliken (capital letter) ; also what is "xc" ?

**Comment by laurent2207** [comment]
Fixed the typos. Do not know wat "xc" is either, so I let it in for now..

**Comment by laurent2207** [comment]
@JKChang2015 You have made the [commit](https://github.com/enanomapper/ontologies/commit/4b7fb97e79b2132ab7dcbef97acda78145344319#). Do you remember why there is "xc" in the Mullikens electronegativity class?

**Comment by JKChang2015** [comment]
@laurent2207 
FYI: NTUA hackathon modelling terms spreadsheet Version 2.0 , 

https://docs.google.com/spreadsheets/d/1F1FPYKkGSlzaMQCfVEvqBoWcMuS-33H9lZX6bzCC-dE/edit#gid=1284797114&range=88:88

**Comment by JKChang2015** [comment]
Deliverable 4.6 

> For the external validation example we used the following model http://test.jaqpot.org:8000/m_detail?name=gaj-10-lr, and the example dataset http://test.jaqpot.org:8000/data_detail?name=xbR5AMG1rOBc  as the test set. The model has been built using the R linear regression algorithm http://test.jaqpot.org:8000/algorithm_detail?name=ocpu-lm on the training set suggested in Gajewicz et al. (10 of 18 published MeOX nanoparticles for predicting HaCaT toxicity, whilst the remaining instances were used as the test set). Two input variables were selected, namely Mullikens electronegativity Xc and Standard enthalpy of formation of metal oxide nanocluster DHcf, whilst the endpoint is described by logLC50. The produced report is provided in the following link: http://test.jaqpot.org:8000/report?name=q8HpVxGYDRFsQLp. The report can be downloaded as a PDF file.

**Comment by laurent2207** [comment]
Okay thanks for this explanation! Will leave the Xc in it (will change the X to be capital).
